### ISSUE TITLE
Upgrading to 0.4.6 breaks Runtime Exposure

### ISSUE THREAD

**Issue by ozgun** [intridea/grape-entity#163]
Here is my entity:

```
expose :my_custom_field

private

def my_custom_field
  1
end
```

After upgrading to 0.4.6, I started to get the following error: 

```
NoMethodError (MyEntity missing attribute my_custom_field on #<MyEntity:0xdc60c70>):
```

If I downgrade to `0.4.5` everything works fine.

Thanks.

**Comment by dblock** [comment]
Does it matter that that method is private (vs. public)?

**Comment by ozgun** [comment]
No, it does not.

**Comment by marshall-lee** [comment]
@dblock It's a regression introduced in #147. This is my fault and not sufficient specs fault. I'll create a `0-4` branch for such issues and fix it soon.

**Comment by marshall-lee** [comment]
This behavior was intended but specs not shown this regression while working on #147.

**Comment by marshall-lee** [comment]
@ozgun thanks for opening this issue! I pushed a fixed 0.4.7 to rubygems.
